backend-lib/src/test/java/cash/z/ecc/android/sdk/internal/fixture/JniAccountBalanceFixture.kt

```diff
@@ -3,7 +3,7 @@ package cash.z.ecc.android.sdk.internal.fixture
 import cash.z.ecc.android.sdk.internal.model.JniAccountBalance
 
 object JniAccountBalanceFixture {
-    const val ACCOUNT_ID: Int = 0
+    val ACCOUNT_UUID: ByteArray = "random_uuid_16_b".toByteArray()
```

Comment:
Ditto: how is this fixture intended to be used? Document it.

Reply:
This test fixture is part of the test package and, thus, not accessible from the SDK business logic. I added a comment to it.